### ISSUE TITLE
Groups not working on lastest master ( grouptree.css / grouptree.fxml not found )

### ISSUE THREAD

**Issue by ajabref** [JabRef/jabref#3053]
Groups have stopped working for me with Jabref 3.8->master.
The groups tab is empty despite the fact that the "groups" field are still present in my bibtex entries.
Both manually allocated groups and dynamic groups are missing
Tested with JabRef version 3.8 to JabRef-4.0-dev--snapshot--2017-07-25--master--0847f5d15.jar on Ubuntu16.02.

Message on the console: 

Launching the jar file without arguments
ERROR StatusLogger Unrecognized format specifier [d]
ERROR StatusLogger Unrecognized conversion specifier [d] starting at position 16 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [thread]
ERROR StatusLogger Unrecognized conversion specifier [thread] starting at position 25 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [level]
ERROR StatusLogger Unrecognized conversion specifier [level] starting at position 35 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [logger]
ERROR StatusLogger Unrecognized conversion specifier [logger] starting at position 47 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [msg]
ERROR StatusLogger Unrecognized conversion specifier [msg] starting at position 54 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [n]
ERROR StatusLogger Unrecognized conversion specifier [n] starting at position 56 in conversion pattern.
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'org.apache.logging.log4j.simplelog.StatusLogger.level' to TRACE to show Log4j2 internal initialization logging.
ERROR StatusLogger Unrecognized format specifier [d]
ERROR StatusLogger Unrecognized conversion specifier [d] starting at position 16 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [thread]
ERROR StatusLogger Unrecognized conversion specifier [thread] starting at position 25 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [level]
ERROR StatusLogger Unrecognized conversion specifier [level] starting at position 35 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [logger]
ERROR StatusLogger Unrecognized conversion specifier [logger] starting at position 47 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [msg]
ERROR StatusLogger Unrecognized conversion specifier [msg] starting at position 54 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [n]
ERROR StatusLogger Unrecognized conversion specifier [n] starting at position 56 in conversion pattern.
AI: INFO 28-07-2017 11:18, 28: Configuration file has been successfully found as resource
File: grouptree.fxml not found, attempting with camel case
Jul 28, 2017 11:18:54 AM javafx.fxml.FXMLLoader$ValueElement processValue
WARNING: Loading FXML document with JavaFX API of version 8.0.112 by JavaFX runtime of version 8.0.60
File: grouptree.css not found, attempting with camel case

**Comment by Siedlerchr** [comment]
I see the only problem here is that your javafx version is out of date:
Please consider installing the latest oracle jdk version

**Comment by lenhard** [comment]
@ajabref Any news here? How does it work out with a newer JDK?

**Comment by tobiasdiez** [comment]
The messages about the missing `grouptree.fxml/css` are harmless and occur always. Moreover, the `StatusLogger` messages should no longer appear in the most recent development version. So sadly the trace is not really help us here.

Do you still have a `groups` comment at the end of the bib file?

**Comment by ajabref** [comment]
Sorry for the long silence, I had a summer break.
@tobiasdiez : the groups comment was gone. I switched several time between different version of Jabref and somehow, it disappeared. I found an old backup where the groups comment was intact and I reinserted it in the bib file by hand and now the issue is gone.
@lenhard : I'm sorry I can't change from the current OpenJDK I'm using as I have other java softs that are quite easily broken when I mess up with java.
